packages/gui/src/components/nfts/detail/NFTDetail.tsx

```diff
@@ -74,7 +74,7 @@ export default function NFTDetail() {
                 </Typography>
 
                 <Typography>
-                  {metadata?.description ?? <Trans>Not Available</Trans>}
+                  {metadata?.description?.split('\n').map((line,i) => <p key={i}>{line}</p>) ?? <Trans>Not Available</Trans>}
```

Comment:
Probably better way is too use pure CSS
sx={{ whiteSpace: 'pre-line'}}

Reply:
I'm not sure how I missed that. Thanks for pointing that out! I've updated it.